spec/models/concerns/courses/edit_options/program_type_spec.rb

```diff
@@ -0,0 +1,55 @@
+require 'rails_helper'
+
+describe Courses::EditOptions::ProgramTypeConcern do
+  let(:example_model) do
+    klass = Class.new do
+      include Courses::EditOptions::ProgramTypeConcern
+
+      def self_accredited?; end
+
+      def provider_is_a_scitt?; end
+    end
+
+    klass.new
+  end
+
+  context 'for a SCITTs self accredited courses' do
+    let(:self_accredited_value) { true }
+    let(:provider_is_a_scitt_value) { 'Y' }
```

Comment:
Is there a reason you have `Y` here, but `false` down below?

Reply:
good spot. Fixed.Thanks